File: deno/src/base/operation_record/operation_record.service.ts

/**
 * 记录系统日志
 */
export async function log(
  input: OperationRecordInput,
) {
  const {
    createOperationRecord,
  } = await import("/gen/base/operation_record/operation_record.dao.ts");
  
  const id = await createOperationRecord(input);
  return id;
}

File: rust/src/src/base/operation_record/operation_record_service.rs
use color_eyre::eyre::Result;

use crate::r#gen::base::operation_record::operation_record_dao;
use crate::r#gen::base::operation_record::operation_record_model::OperationRecordInput;

#[allow(dead_code)]
pub async fn log(
  input: OperationRecordInput,
) -> Result<()> {
  operation_record_dao::create(
    input,
    None,
  ).await?;
  Ok(())
}
